packages/pluggableWidgets/sidebar-web/src/Sidebar.editorConfig.ts

```diff
@@ -0,0 +1,37 @@
+import {
+    hidePropertiesIn,
+    Properties,
+    StructurePreviewProps,
+    transformGroupsIntoTabs
+} from "@mendix/piw-utils-internal";
+
+import { SidebarPreviewProps } from "../typings/SidebarProps";
+
+export function getProperties(
+    values: SidebarPreviewProps,
+    defaultProperties: Properties,
+    platform: "web" | "desktop"
+): Properties {
+    if (values.toggleMode === "none" && values.toggleMode === "push" && values.toggleMode === "slideOver") {
```

Comment:
As @tamelbrun mentioned, my previous code suggestion was incorrect as it should be an OR:

```suggestion
    if (values.toggleMode === "none" || values.toggleMode === "push" || values.toggleMode === "slideOver") {
```